info.json

```diff
@@ -14,7 +14,7 @@
         "? bobplates",
         "? bobwarfare",
         "? AtomicRocketScreenBurn",
-        "? IndustrialRevolution >= 2.0.0",
+        "? IndustrialRevolution >= 3.1.5",
```

Comment:
Everywhere else IR3 is referenced as IndustrialRevolution3, here it is just IndustrialRevolution.

Reply:
Updated info.json, the typo was there. IndustrialRevolution3 is the correct mod name.